src/org/loklak/objects/MessageEntry.java

```diff
@@ -57,7 +57,7 @@
     // left boundary must be space since the @ is itself a boundary
     public final static Pattern USER_PATTERN = Pattern.compile("(?:[ (]|^)(@..*?)(?:\\b|$)");
     // left boundary must be a space since the # is itself a boundary
-    final static Pattern HASHTAG_PATTERN = Pattern.compile("(?:[ (]|^)(#..*?)(?:\\b|$)");
+    final static Pattern HASHTAG_PATTERN = Pattern.compile("(?:[ (]|^)#(..*?)(?:\\b|$)");
```

Comment:
Any tests? Doesn't keeping the `#` outside mandate that the hash is available in the pattern?

It'd always be better to have a test to pass and fail these patterns, that'd give a detailed reason as to why this change in regex is needed.